src/commands/deploy/deploy.ts

```diff
@@ -101,11 +103,13 @@ export async function deploy(context: IActionContext, confirmDeployment: boolean
     siteConfig = siteConfig ? siteConfig : await node.root.client.getSiteConfig();
 
     if (javaUtils.isJavaRuntime(siteConfig.linuxFxVersion)) {
-        const javaArtifactFiles: vscode.Uri[] = await workspaceUtil.findFilesByFileExtension(deployContext.deployFsPath, javaUtils.getArtifactTypeByJavaRuntime(siteConfig.linuxFxVersion));
-        if (javaArtifactFiles.length > 0) {
-            const javaArtifactQp: IAzureQuickPickItem<string>[] = workspaceUtil.mapFilesToQuickPickItems(javaArtifactFiles);
-            // check if there is a jar/war file in the fsPath that was provided
-            deployContext.deployFsPath = <string>(await ext.ui.showQuickPick(javaArtifactQp, { placeHolder: `Select the ${javaUtils.getArtifactTypeByJavaRuntime(siteConfig.linuxFxVersion)} file to deploy...` })).data;
+        if (!fileExtensions) {
```

Comment:
I don't think we need the check or any code inside of it anymore.  This was originally used to prompt the user a second time for a .jar/.war file IF we detected that they were deploying to a Java app.  We do still need the `javaUtils.configureJavaSEAppSettings` though